components/atom/input/src/Input/Features/WithIcon/index.scss

```diff
@@ -0,0 +1,31 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$r-atom-input-password-toggle-button: $gutter !default;
+
+.sui-AtomInput-withIcon {
+  position: relative;
+  width: 100%;
+
+  .sui-AtomInput-input {
+    padding-left: 50px;
+  }
+
+  &--icon {
+    align-items: center;
+    color: $c-primary;
+    cursor: pointer;
+    display: flex;
+    height: 25px;
```

Comment:
what about using sui-theme icon sizes?
https://github.com/SUI-Components/sui-theme/blob/master/src/components/_icon.scss

btw, @jcger  should we use icon sizes related to the input size?

Reply:
we are using https://github.com/SUI-Components/sui-theme/blob/master/src/settings/_size.scss for icon sizes. (hope the sui-theme refactor happens soon). If related or not should be defined by UX, looks light they only need one icon size at the moment